torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2179,6 +2179,15 @@ def _param_fqns(self) -> Iterator[Tuple[str, str, str]]:
             fqn = f"{module_name}{param_name}"
             yield fqn, param_name, module_name
 
+    @property
+    def _shared_param_fqns(self) -> Iterator[Tuple[str, str, str]]:
+        for param_name, module_name in (
+            self._fsdp_wrapped_module.handle.shared_parameter_module_names()
```

Comment:
nit: i remembered _fsdp_wrapped_module could have multiple handles, so this should be "self._fsdp_wrapped_module.handles[0]"? @awgu